src/frontend/apps/markdown/MdxRenderingOptionsSelector/index.spec.tsx

```diff
@@ -0,0 +1,103 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { wrapInIntlProvider } from 'utils/tests/intl';
+
+import MdxRenderingOptionsSelector from '.';
+
+describe('<MdxRenderingOptionsSelector />', () => {
+  it('send options change', async () => {
+    window.scrollTo = jest.fn(); // required to test, see grommet
+
+    const setRenderingOptions = jest.fn();
+
+    const { rerender } = render(
+      wrapInIntlProvider(
+        <MdxRenderingOptionsSelector
+          renderingOptions={{ useMdx: false }}
+          setRenderingOptions={setRenderingOptions}
+        />,
+      ),
+    );
+
+    const dropButton = screen.getByRole('button', { name: 'Settings' });
+    userEvent.click(dropButton); // open the menu
+
+    // Toggle MDX
+    const toggleMdxButton = screen.getByTestId('mdx-options-toggle-mdx');
+
+    expect(screen.getByText('MDX disabled')).toBeInTheDocument();
```

Comment:
```suggestion
    screen.getByText('MDX disabled'));
```
If `MDX disable` is not present in the document, your test will fail, so there is non eed to use `ToBeInTheDocument` here.